CodeStarterCamp_Week2/main.swift

```diff
@@ -1,12 +1,26 @@
-//
-//  main.swift
-//  CodeStarterCamp_Week2
-//
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
-//
+func makeWinningLottoNumbers() -> [Int] {
+    var winningLottoNumbers = Set<Int>()
+    while winningLottoNumbers.count < 6 {
+        winningLottoNumbers.insert(Int.random(in: 1...45))
+    }
+    return winningLottoNumbers.sorted()
+}
 
-import Foundation
-
-print("Hello, World!")
+func checkMyLottoNumbers(myLottoNumbers: [Int]) {
+    let winningLottoNumbers = makeWinningLottoNumbers()
+    print("이번 당첨번호: \(winningLottoNumbers)")
+    var sameNumbers: [Int] = []
+    
+    for nums in myLottoNumbers {
+        if winningLottoNumbers.contains(nums) == true {
+            sameNumbers.append(nums)
+        }
+    }
+    if sameNumbers.count >= 1 {
+        print("축하합니다! 겹치는 번호는 \(sameNumbers.map {String($0)}.joined(separator: ",")) 입니다!")
+    } else if sameNumbers.count == 0 {
```

Comment:
요것도 참고하시라고 말씀 드립니다! 갯수가 0개인지 확인하는 프로퍼티 `isEmpty`를 활용하는 방법도 있습니다.
```suggestion
    } else if sameNumbers.isEmpty {
```

그리고 흐름상 이 조건문 자체가 필요하지 않을 수도 있겠네요! 😅